app/assets/stylesheets/views/signin.scss

```diff
@@ -51,6 +51,10 @@
     font-size: var(--fs-2xl);
     line-height: var(--lh-tight);
 
+    // overflow-wrap: anywhere isn't supported in Safari, in which case this fallback applies
+    overflow-wrap: break-word;
+    overflow-wrap: anywhere;
```

Comment:
Refactor (non-blocking): Consider a mixin for the time being since the same thing is used in 11 places. Maybe it's overkill. 🤔